package.json

```diff
@@ -33,11 +32,9 @@
     "@types/node": "^22.0.0",
     "awilix": "^10.0.2",
     "c8": "^10.1.2",
-    "eslint": "^9.5.0",
-    "eslint-config-prettier": "^9.1.0",
-    "eslint-plugin-prettier": "^5.1.3",
     "fastify": "^4.26.2",
-    "prettier": "^3.2.5",
+    "snazzy": "^9.0.0",
+    "standard": "^17.1.0",
```

Comment:
why not neostandard?

Reply:
One dep to rule them all: when we have these silly trouble due a breaking change, it is always a pain if we use one different library for every repo. Pain means time that nobody wants to invest in these tasks. 

This pr is no brainier because we run these deps on every other fastify repo. We know them, we trust them. 

Evaluating something else is something that we need to do (standard is no more used in the main repo), but it is a different task imho

Reply:
sounds good!